src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -47,7 +47,13 @@
     public function load(/*...$paths*/)
```

Comment:
Well, it does not really pollute the method signature if we *really* want to require at least 1 arg. In PHP 5.6+, ``public function load(...$paths)`` means 0+ arguments, not 1+.